config.py

```diff
@@ -132,6 +132,9 @@ class Config(object):
     STATSD_HOST = "statsd.hostedgraphite.com"
     STATSD_PORT = 8125
 
+    REDIS_ENABLED = False
+    REDIS_URL = "redis://localhost:6379/0"
```

Comment:
It will be. I'll do another pull request that switches it on, and handles all the configs. I want to hand hold it in preview first.